src/otx/core/data/transform_libs/torchvision.py

```diff
@@ -3921,3 +3921,44 @@ def _dispatch_transform(cls, cfg_transform: DictConfig | dict | tvt_v2.Transform
             raise TypeError(msg)
 
         return transform
+
+
+class RandomIoUCrop(tvt_v2.RandomIoUCrop):
```

Comment:
If we use already defined RandomIOUCrop in this file, the performance issues occur?
![image](https://github.com/user-attachments/assets/0d808215-dfe2-4847-bcfc-9638fbaa70ab)